online-shopping-website/frontend/src/Components/Seller/SellerProductsForm.js

```diff
@@ -289,67 +302,70 @@ export const AddNewProductForm = () => {
 
     return (
         <form onSubmit={handleSubmit}>
-            <Stack spacing={2} sx={{ maxWidth: '550px', margin: 'auto' }}>
-                <h1>Add a product</h1>
-                <TextField 
+            <Stack spacing={2} sx={{maxWidth: '550px', margin: 'auto'}}>
+                <h1 className="TextGreen">Add a product</h1>
+                <TextField
                     name="name"
-                    label="Name" 
+                    label="Name"
                     required
                     value={newProduct.name}
-                    onChange={handleFieldChange} 
-                    />
-                <TextField 
+                    onChange={handleFieldChange}
+                />
+                <TextField
                     name="price"
                     label="Price"
                     type="number"
-                    required 
+                    required
                     value={newProduct.price}
-                    onChange={handleFieldChange} 
+                    onChange={handleFieldChange}
                     InputProps={{
-                        inputProps: { min: 0, step: "0.5" },
+                        inputProps: {min: 0, step: "0.5"},
                         endAdornment: <InputAdornment position="end">Ɖ</InputAdornment>
                     }}
                 />
-                <TextField 
+                <TextField
                     name="description"
-                    label="Description" 
-                    required 
+                    label="Description"
+                    required
                     value={newProduct.description}
-                    onChange={handleFieldChange} 
-                    multiline 
```

Comment:
Try to avoid these spacing changes, it just bloats the line changes on the PR. Not a big deal, just for the future!

Reply:
Prettier keeps doing this on save. Gotta fix my settings in VS Code😅